exhibitions/src/main/kotlin/edu/artic/exhibitions/ExhibitionDetailFragment.kt

```diff
@@ -0,0 +1,202 @@
+package edu.artic.exhibitions
+
+import android.animation.ValueAnimator
+import android.content.Intent
+import android.graphics.drawable.Drawable
+import android.net.Uri
+import android.os.Bundle
+import android.support.transition.TransitionInflater
+import android.view.View
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.DataSource
+import com.bumptech.glide.load.engine.GlideException
+import com.bumptech.glide.request.RequestListener
+import com.bumptech.glide.request.RequestOptions
+import com.bumptech.glide.request.target.Target
+import com.fuzz.rx.bindToMain
+import com.fuzz.rx.disposedBy
+import com.jakewharton.rxbinding2.view.clicks
+import com.jakewharton.rxbinding2.widget.text
+import edu.artic.db.models.ArticExhibition
+import edu.artic.viewmodel.BaseViewModelFragment
+import edu.artic.viewmodel.Navigate
+import kotlinx.android.synthetic.main.fragment_exhibition_details.*
+import kotlinx.android.synthetic.main.fragment_exhibition_details.view.*
+import timber.log.Timber
+import kotlin.reflect.KClass
+
+
+class ExhibitionDetailFragment : BaseViewModelFragment<ExhibitionDetailViewModel>() {
+
+    override val viewModelClass: KClass<ExhibitionDetailViewModel>
+        get() = ExhibitionDetailViewModel::class
+    override val layoutResId: Int
+        get() = R.layout.fragment_exhibition_details
+    override val title: String
+        get() = ""
+
+    private val exhibition by lazy { arguments!!.getParcelable<ArticExhibition>(ARG_EXHIBITION) }
```

Comment:
`requireArguments()` instead. also if its argument handling you need, safe args to the rescue. https://developer.android.com/topic/libraries/architecture/navigation/navigation-implementing#Safe-args